src/client/vite.config.ts

```diff
@@ -281,11 +302,24 @@ const setConfig = ({ mode }) => {
   }
 
   if (TARGET === "openfin" || TARGET === "launcher") {
-    plugins.push(copyOpenfinPlugin(isDev, TARGET))
+    plugins.push(copyOpenfinPlugin(isDev, openfinBaseUrl, TARGET))
   }
 
+  let proxy
   if (process.env.VITE_MOCKS) {
     plugins.unshift(apiMockReplacerPlugin())
+  } else {
```

Comment:
TS is complaining about the ambiguous typing of `proxy`. Perhaps we can improve this by removing the else clause and instead do something like
```
const proxy = process.env.VITE_MOCKS ? null : {
  ...
}
```

Reply:
yep, that works .. as long as I use `undefined` as the fallback for when in mocks mode
(note to ourselves, mock mode is broken right now ... so this needs to be on our Tech Debt list)